specification/signalr/resource-manager/Microsoft.SignalRService/preview/2020-07-01-preview/signalr.json

```diff
@@ -1466,6 +1550,32 @@
       ],
       "properties": {}
     },
+    "SignalRTlsSettings": {
```

Comment:
is it necessary to name it as SignalRTlsSettings given it's already under SignalR service namespace

Reply:
Since our other class names contain `SignalR`, I would like to keep the same naming convention